src/config/fabric-icons.json

```diff
@@ -15,29 +23,21 @@
       "unicode": "EE69"
     },
     {
-      "name": "Table",
```

Comment:
Can we append to this instead of replacing/rearranging every time. I can't tell what's being added and deleted

Reply:
Missing this icon:
![image](https://user-images.githubusercontent.com/60453211/86079181-c7273780-ba44-11ea-88bc-c27a09fe6e33.png)

If we append instead of replacing/rearranging whole file, catching these errors will be easier

Reply:
This one is the same, you told me that we have to replace this file. I'll add the missing icon though. What is the name of the missing icon?

Reply:
fixed